tp/containers/k8s-kubeadm-intro.fr.md

```diff
@@ -119,18 +122,18 @@ EOF
 
 Sur les machines, 2 interfaces réseaux sont présents :
 
-- eth0: nécessaire à Vagrant
-- eth1: le réseau privé que nous allons utiliser pour kubeadm
+- enp0s3: nécessaire à Vagrant
```

Comment:
Is this always the case ? does it depends on the vagrantbox used or the vagrant version ?

Reply:
I'm not sure about the consistence. But for the past few months, I had enp0s3/8 each time. Anyway, the TP specifies that the name can change et such changes should be taken into consideration. At least with enp0s3/8, we have a common default choice.

Reply:
AFAIK it depends on the vagrantbox, starting from Ubuntu 16 the default interfaces are usually `enp0s3` and `enp0s8`
I've tested on Ubuntu 14 (`eth0` and `eth1`), Ubuntu 16+ (`enp0s3` and `enp0s8`)

(source: [systemd v197's predictable interface names](https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/), Ubuntu versions prior to 14.10+ seems to be using Upstart)